Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -1,8 +1,34 @@
 let order = [
   { itemName: "Hot cakes", quantity: 1, unitPricePence: 232 },
+  { itemName: "Falafel with salad", quantity: 1, unitPricePence: 224},
   { itemName: "Apple Pie", quantity: 2, unitPricePence: 139 },
   { itemName: "Egg McMuffin", quantity: 1, unitPricePence: 280 },
   { itemName: "Sausage McMuffin", quantity: 1, unitPricePence: 300 },
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+let sortedOrder = order.sort((a, b) => a.itemName.length - b.itemName.length);
```

Comment:
If you don't re-assign the value of sortedOrder later in the code, what could you change 'let' to?